### ISSUE TITLE
poll radiothon status from donormotor

### ISSUE THREAD

**Issue by echarlie** [wuvt/wuvt-site#397]
since the donation app needs to know radiothon mode for donation handling, and will have the UI for editing those redis values there, we should poll it.

**Comment by mutantmonkey** [comment]
We'll want to make sure we cache the status for a sufficient amount of time (maybe 5 minutes or something) so we don't unnecessarily impact performance on either end. Ideally we'd perform the cache update asynchronously as well; we probably don't want to slow page loads for this.

**Comment by echarlie** [comment]
yes. we should shove it in redis as we do now. Celery is the way we're doing asynchronous tasks in wuvt-site, yes?

**Comment by echarlie** [comment]
donormotor currently exposes radiothon status as /api/v1/radiothon, which returns {"radiothon": "true"} (or false). it is an incredibly naïve API.
